### ISSUE TITLE
About Page: Content should be contained in a landmark region

### ISSUE THREAD

**Issue by davefollett** [codingblocks/qit.cloud#228]
## 💥 Impact rating
moderate

## 👣 Steps to reproduce
Navigate to http://localhost:3000/about/ and [run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test).

## ⚛️ Element(s) with issue
`<html lang="en">`

## 🛠️ Recommended fix
Content not contained by an ARIA landmark

Related nodes:
1. `h2`
1. `p:nth-child(2)`
1. `.finePrint`
1. `button`

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/region?application=AxeChrome

**Comment by davefollett** [comment]
This is what needs to be done to fix this issue, QIT also has this on most of its pages.

![about-a11y](https://user-images.githubusercontent.com/6683520/66722466-258b8780-eddc-11e9-8cb5-d6cc9455450a.png)

**Comment by davefollett** [comment]
Fixed by PRs #329 and #330 by @MinimumViablePerson
